mantidimaging/core/io/saver.py

```diff
@@ -241,7 +237,6 @@ def _nexus_save(nexus_file: h5py.File, dataset: StrictDataset, sample_name: str)
     # data field
     data = tomo_entry.create_group("data")
     _set_nx_class(data, "NXdata")
-    data["data"] = detector["data"]
     data["rotation_angle"] = rotation_angle
     data["image_key"] = detector["image_key"]
```

Comment:
Can these move into the `processed-data` so that they are in the same place relative to the data as before.